README.md

```diff
@@ -1,7 +1,11 @@
 bodylabs-javascript-style
-=============
+=========================
 
-Body Labs JavaScript style, using JSCS and eslint.
+Body Labs JavaScript style, using eslint.
+
+This is provided as an eslint module, because it lets us bundle together
+multiple configs, and also allows us to provide code for our own rules,
+should we add any in the future.
```

Comment:
Do you want to add a short example of using this as a plugin, like you did in the PR description?